docs/source/installation/translations.rst

```diff
@@ -106,38 +113,6 @@ So if by any reason it doesn't work for you, feel free to :ref:`ask us <contact>
 .. _zh_Hant_TW: https://www.localeplanet.com/icu/zh-Hant-TW/index.html
 
 
-Contributing
-============
-
-As a **translator**, you should have a good knowledge of the Indico functions
-(from the user side at least). Then you can subscribe to the abovementioned
-`Transifex site for Indico <https://www.transifex.com/indico/>`_
-and request membership of one of the translation teams. You should also contact
-the coordinators; some languages have specific coordinators assigned.
-They may point you to places, where work is needed and which rules have
-been agreed for the translations.
-
-The glossary is usually of big help to obtain a uniform translation of all
-technical terms. Use it!
-
-As a **programmer** or **developer**, you will have to be aware of the needs and
-difficulties of translation work.
-A `Wiki page for Internationalisation <https://github.com/indico/indico/wiki/Internationalisation>`_
-is available from github (slightly outdated and we should eventually move it to this documentation).
-It describes the interface between translating and programming and some conventions to be followed.
-Everyone involved in translating or programming Indico should have read it before starting the work.
```

Comment:
As this text itself suggests, I'd eventually like to turn it into a separate devguide